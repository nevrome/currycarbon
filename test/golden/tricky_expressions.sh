# tricky expressions
## barely overlapping product
currycarbon "uncalC14(3200,30) * rangeBP(3000,2800)"
