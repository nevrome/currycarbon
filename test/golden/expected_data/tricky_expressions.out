currycarbon v0.4.0.2 (UTF-8)
Method: Bchron {distribution = StudentTDist {ndf = 100.0}}
Curve: IntCal20
Calibrating...
Done.
CalEXPR: [1] (1p:3200±30BP * 1P:3000BP-2800BP)
Calibrated: 1049BC >> 1049BC > 1025BC < 1019BC << 1017BC
1-sigma: 1049-1048BC, 1026-1019BC
2-sigma: 1049-1036BC, 1033-1033BC, 1029-1017BC


                      ▁                                          
                      ▒▁                                         
                     ▁▒▒                                         
         ▁           ▒▒▒▁                                        
         ▒          ▁▒▒▒▒                                        
         ▒▁▁▁▁▁▁  ▁▁▒▒▒▒▒▁                                       
         ▒▒▒▒▒▒▒▁▁▒▒▒▒▒▒▒▒▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁
 -1050 ┄────────────────────────┬───────────────────────────────┄ -940
    BC   >           ^  <<                                        BC
         ─          ─────                                        
         ─────── ─ ───────                                       
